halo2_backend/src/plonk/prover.rs

```diff
@@ -216,12 +232,19 @@ impl<
                     instance_polys,
                 })
             };
-        let instance: Vec<InstanceSingle<Scheme::Curve>> = instances
+
+        // Get the polynomials of all circuits instances
+        // [TRANSCRIPT-2]
```

Comment:
Can you explain the meaning of these references?
Ah, after seeing the verifier: this are references to match the prover and verifier transcripts writes - reads right?